.travis.yml

```
@@ -16,6 +16,9 @@ cache:
 before_install:
   - travis_retry composer self-update
 
+before_script:
+  - cp .env.example .env
+
```

This should fix the failing test:

```suggestion
- php artisan key:generate

```

---
Please also move this `before_script` section to just before the `script` section in the file.